webpack/helpers.js

```diff
@@ -4,6 +4,31 @@ function root(path) {
   return resolve(__dirname, '..', path);
 }
 
+function checkNewVersion(options, appVersion){
+  let http = require('http');
+  let semver = require('semver');
+  let notifier = require('node-notifier');
+
+  http.get(options, function(resp){
+    let body = '';
+    resp.on('data', function(chunk){
+      body += chunk;
+    });
+    resp.on('end', function(){
+      let parsed = JSON.parse(body);
+      if (semver.gt(parsed.message, appVersion)){
+        notifier.notify({
+          'title': `New version is available!`,
+          'message': `bterm version ${parsed.message} is available.`
+        });
+      }
+    });
+  }).on("error", function(e){
+    console.log("Got error: " + e.message);
```

Comment:
please cleanup console log for the production, there has to be logging system in i.e. files...